test_runner/fixtures/pageserver/allowed_errors.py

```diff
@@ -106,6 +106,11 @@ def scan_pageserver_log_for_errors(
     ".*delaying layer flush by \\S+ for compaction backpressure.*",
     ".*stalling layer flushes for compaction backpressure.*",
     ".*layer roll waiting for flush due to compaction backpressure.*",
+    # Low level HTTP server errors can happen during restart/shutdown.
+    ".*Failed to accept TCP connection.*",
+    ".*Failed to accept TLS connection.*",
+    ".*Failed to serve HTTPS connection.*",
+    ".*Failed to serve HTTP connection.*",
```

Comment:
Are these very noisy? I don't have a good sense of what they'll log and how they can fail, but if they end up spamming a ton of connection closed errors and such we should probably either handle and silence those errors or downgrade to info as appropriate.

Reply:
Quite noisy during test teardown, that's why I initially used `debug!`. Switched to `info!`

Reply:
What does it mean `can also match on the error`? Do you mean manually checking for some well-known errors and avoid logging them?

Reply:
Yes. We frequently do this elsewhere.

Reply:
I think the two main ones that we don't care about are `std::io::ErrorKind::BrokenPipe` and `std::io::ErrorKind:: UnexpectedEof`. But you can just look at the test errors that were allowed here.